OPENCODE_INSTALL.md

```
@@ -28,7 +28,18 @@ Verify by asking: "Tell me about your Development Crew"
 
 ## Usage
 
-Use OpenCode's native `skill` tool:
+Use the skill-derived slash commands when available:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)